src/pages/Login/Login.scss

```diff
@@ -94,30 +94,28 @@
       max-width: 750px;
       width: 80%;
       left: 11%;
-      top: 125px;
+      top: 150px;
       background-color: #f8d7da;
       color: #fd1015;
       display: flex;
       align-items: center;
       justify-content: space-around;
       border-radius: 8px;
       box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
-      animation: pop 800ms ease-out;
+      animation: pop 300ms ease-in-out;
       animation-fill-mode: forwards;
       min-height: 50px;
       padding: 8px;
     }
 
     @keyframes pop {
-      0% {
+      from {
         display: block;
         top: 100px;
         opacity: 1;
       }
-      25% {
-        opacity: 1;
-      }
-      90% {
+      to {
+        top: 150px;
         opacity: 1;
       }
     }
```

Comment:
I understand this is something you worked on previously and saw space for improvement, however try to stick to the pages you are working on to make sure there is no cross editing of areas and notify the PO when refactoring work from a previous ticket for reasons stated.